templates/navigation/menu.html.twig

```diff
@@ -25,7 +25,7 @@
         %}
         <ul{{ attributes.addClass(menu_classes) }}>
       {% else %}
-        <ul class="nav__subnav">
```

Comment:
Isn’t the subnav a sub-component of the nav component? I can’t imagine that I would ever use .nav--subnav as a standalone variant. I would always have that nested under the parent menu component.